### ISSUE TITLE
Does not work with File Fields. Upon editing the Resource, the File field is reset to null.

### ISSUE THREAD

**Issue by 949mac** [dcasia/conditional-container#45]
**Steps to Reproduce:**

(1) Setup something like:

```
ConditionalContainer::make([
   File::make('Proof of Service', 'current_proof_of_service')->updateRules('file', 'mimes:pdf')
])->if('has_current_proof_of_service truthy true'),
```

(2) Update the form. When you go back to the detail view, observe you can see the file was uploaded.

(3) Click on Edit, and save w/out updating the file.

(4) ❗ Observe the file has been removed an is now NULL.

**Comment by CptIceberg** [comment]
I encountered the same bug, which is pretty annoying. 
I resolved it with this workaround, which is by no mean pretty, and make a mess in the storage, but it works if you don't have much entries to save. It is a method in the observer of the model :
```php
 public function updating(Template $template): void
    {
        if ($template->content_type === 'pdf') {
            $path = "pdf_path_$template->id" ;

            if (Storage::disk('private')->exists($path)) {
                $content = Storage::disk('private')->get($path);
                if ($content === $template->pdf_path || ! $template->pdf_path) {
                    $template->pdf_path = $content;
                    $template->saveQuietly();
                }

                return;
            }
            Storage::disk('private')->put($path, $template->pdf_path);
        }
    }

**Comment by milewski** [comment]
Fixed on version [v1.4.1](https://github.com/dcasia/conditional-container/releases/tag/v1.4.1)
